_migrations/getting-started-data-migration.md

```diff
@@ -0,0 +1,329 @@
+---
+layout: default
+title: Quickstart - Data migration
+nav_order: 10
+---
+
+# Getting started: Data migration
+
+This document outlines how to deploy the Migration Assistant and execute an existing data migration using `reindex-from-snapshot` (RFS). It uses AWS for the sake of illustration. However, the steps can be modified for use with other cloud providers.
+
+Note that this does not include steps for deploying and capturing live traffic, which is necessary for a zero-downtime migration.  
+{: .note}
+
+## Prerequisites and assumptions
+
+Before utilizing this quickstart, make sure you fulfill the following prerequisites:
+
+* Verify your that migration path [is supported](https://opensearch.org/docs/latest/migrations/is-migration-assistant-right-for-you/#supported-migration-paths). Note that we test with the exact versions specified, but you should be able to migrate data on alternative minor versions as long as the major version is supported.
+* The source cluster must be deployed with the S3 plugin.
+* The target cluster must be deployed.
+
+Using this quickstart assumes the following:
+
+* A snapshot will be taken and stored in S3 in this guide, and the following assumptions are made about this snapshot:
+  * The `_source` flag is enabled on all indices to be migrated.
+  * The snapshot includes the global cluster state (`include_global_state` is `true`).
+  * Shard sizes up to approximately 80GB are supported. Larger shards will not be able to migrate. If this is a blocker, please consult the migrations team.
```

Comment:
```suggestion
  * Shard sizes up to approximately 80GB are supported. Larger shards can not be migrated. If this is blocker for your migration, contact the migration's team.
```

Reply:
@peternied: do we have a link with contact information for the migrations team? Should I direct them to the migrations repo?

Reply:
Directing them to the repo is a good option.